qa/rpc-tests/fundrawtransaction.py

```diff
@@ -575,7 +601,7 @@ def run_test(self):
         outputs = {self.nodes[2].getnewaddress() : watchonly_amount / 2}
         rawtx = self.nodes[3].createrawtransaction(inputs, outputs)
 
-        result = self.nodes[3].fundrawtransaction(rawtx, True)
+        result = self.nodes[3].fundrawtransaction(rawtx, {'includeWatching': True })
```

Comment:
Can you leave one of them and mention that this is testing backward compatibility?